hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/backend/AbstractBackendEntityEnricher.java

```diff
@@ -256,22 +256,35 @@ protected Map<String, org.hypertrace.core.datamodel.AttributeValue> getAttribute
     return attributes;
   }
 
+  protected Entity mergeBackendEntities(Entity existingEntity, Entity newEntity) {
```

Comment:
Can we add a comment that we can replace the `mergeBackendEntity` flow with an entity update flow?